src/Models/ArduinoDeviceBase.ts

```diff
@@ -162,10 +163,10 @@ export abstract class ArduinoDeviceBase implements Device {
     await this.generateCppPropertiesFile(createTimeScaffoldType, board);
 
     // Configurate device environment
-    const res = await this.configDeviceEnvironment(
+    await this.configDeviceEnvironment(
         this.deviceFolder, createTimeScaffoldType);
 
-    return res;
+    return true;

```

Comment:
It seems that the function always returns true if there is no exception, then the return value is not necessary. Can we make the function return void instead of boolean?

Reply:
Fixed.